lib/template.html

```diff
@@ -35,6 +35,20 @@
   {% assign seo_description = seo_description | markdownify | strip_html | strip_newlines | escape_once %}
 {% endif %}
 
+{% if page.author %}
+  {% if page.author.name %}
+    {% assign seo_author_name = page.author.name %}
+  {% else %}
+    {% assign seo_author_name = page.author %}
+  {% endif %}
+
+  {% if page.author.twitter %}
+    {% assign seo_author_twitter = page.author.twitter | replace:"@","" | prepend:"@" %}
+  {% else %}
+    {% assign seo_author_twitter = page.author | replace:"@","" | prepend:"@" %}
```

Comment:
> It looks like you're adding the `@`?

[It was there before.](https://github.com/benbalter/jekyll-seo-tag/pull/26/files#diff-b18f6b02baa17fff6667f45aca6143c5L101)

The [example at the bottom of this page](https://dev.twitter.com/cards/getting-started) includes the `@`